projects/ziti-console-lib/src/lib/reset-enrollment/reset-enrollment.component.ts

```diff
@@ -0,0 +1,86 @@
+/*
+    Copyright NetFoundry Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
+
+import {Component, Inject, ViewChild} from '@angular/core';
```

Comment:
whole new file shouldn't be part of this PR?

Reply:
This was a duplicate file I removed in my last PR. Might have gotten added back in as part of conflict resolution

@edthompson-nf

Reply:
removed